backend/server.py

```
@@ -1,561 +1,741 @@
+        input_questions = _require_list(data.get("input_question"), "input_question")
+        input_options = _require_list(data.get("input_options"), "input_options")
+
+        if not input_questions or not input_options or len(input_questions) != len(input_options):
+            raise InvalidRequest(
+                "'input_question' and 'input_options' must be non-empty lists of the same length."
+            )
+
+        qa_model = _get_service("qa_model")
+        outputs = []
+
+        for question, options in zip(input_questions, input_options):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)